sandbox/dandi.bib

```diff
@@ -0,0 +1,4058 @@
+  title = {MICrONS Two Photon Functional Imaging},
+  publisher = {DANDI Archive},
+  year = {2023}
+}
+
+
+# Take latest as the default
+@misc{dandi.000402,
+  doi = {10.48324/DANDI.000402/0.230307.2132},
+  url = {https://dandiarchive.org/dandiset/000402/0.230307.2132},
+  author = {Bae, J. Alexander and Baptiste, Mahaly and Bodor, Agnes L. and Brittain, Derrick and Buchanan, JoAnn and Bumbarger, Daniel J. and Castro, Manuel A. and Celii, Brendan and Cobos, Erick  and Collman, Forrest and Maçarico da Costa, Nuno and Dorkenwald, Sven and Elabbady, Leila and Fahey, Paul G. and Fliss, Tim and Froudarakis, Emmanouil and Gager, Jay and Gamlin, Clare and Halageri, Akhilesh and Hebditch, James and Jia, Zhen and Jordan, Chris and Kapner, Daniel and Kemnitz, Nico and Kinn, Sam and Koolman, Selden and Kuehner, Kai and Lee, Kisuk and Li, Kai and Lu, Ran and Macrina, Thomas and Mahalingam, Gayathri and McReynolds, Sarah and Miranda, Elanine and Mitchell, Eric and Mondal, Shanka Subhra and Moore, Merlin and Mu, Shang and Muhammad, Taliah and Nehoran, Barak and Ogedengbe, Oluwaseun and Papadopoulos, Christos and Papadopoulos, Stelios and Patel, Saumil and Pitkow, Xaq and Popovych, Sergiy and Ramos, Anthony and Reid, R. Clay and Reimer, Jacob and Schneider-Mizell, Casey M. and Seung, H. Sebastian and Silverman, Ben and Silversmith, William and Sterling, Amy and Sinz, Fabian H. and Smith, Cameron L. and Suckow, Shelby and Takeno, Marc and Tan, Zheng H. and Tolias, Andreas S. and Torres, Russel and Turner, Nicholas L. and Walker, Edgar Y. and Wang, Tianyu and Williams, Grace and Williams, Sarah and Willie, Kyle and Willie, Ryan and Wong, William and Wu, Jingpeng and Xu, Chris and Yang, Runzhe and Yatsenko, Dimitri and Ye, Fei and Yin, Wenjing and Yu, Szi-chieh},
+  title = {MICrONS Two Photon Functional Imaging},
+  publisher = {DANDI Archive},
+  year = {2023}
+}
+
+
+# DANDISET 000404
+@misc{dandi.000404/0.230605.2024,
+  doi = {10.48324/DANDI.000404/0.230605.2024},
+  url = {https://dandiarchive.org/dandiset/000404/0.230605.2024},
+  author = {Athalye, Vivek R and Khanna, Preeya and Gowda, Suraj and Orsborn, Amy L and Costa, Rui M and Carmena, Jose M},
+  keywords = {neural population dynamics, motor cortex, motor control, brain-machine interface, neuroprosthetics, optimal feedback control, motor commands, movement representations, dynamical systems},
+  title = {Monkey 2D cursor BMI},
```

Comment:
apparently `data.crosscite.org` is not doing that great of a job here -- for BibTeX it ideally should have made it into

```suggestion
  title = {Monkey {2D} cursor {BMI}},
```

to prevent LaTeX lower casing them... now we just need to figure out where is that code to file a complaint :-/

Reply:
the easiest of cause IIRC is just to double `{}` it ie
 ```suggestion
  title = {{Monkey 2D cursor BMI}},
```
and in principle we could do that ourselves here after someone confirms that my concern here is correct 

attn @TheChymera as who might know and @bendichter as who might care ;-)